client/web/src/enterprise/insights/components/insights-view-grid/components/insight-context-menu/InsightContextMenu.tsx

```diff
@@ -61,14 +63,26 @@ export const InsightContextMenu: React.FunctionComponent<InsightCardMenuProps> =
                             </MenuLink>
 
                             {showYAxisToggleMenu && (
-                                <MenuLink
+                                <MenuItem
+                                    role="menuitemcheckbox"
                                     data-testid="InsightContextMenuEditLink"
-                                    className={classnames('btn btn-outline border-bottom', styles.item)}
-                                    onClick={onToggleZeroYAxisMin}
+                                    className={classnames(
+                                        'd-flex align-items-center justify-content-between',
+                                        styles.item
+                                    )}
+                                    onSelect={onToggleZeroYAxisMin}
+                                    aria-checked={zeroYAxisMin}
                                 >
-                                    <CheckIcon size={16} className={classnames('mr-2', { 'd-none': !zeroYAxisMin })} />{' '}
-                                    Start Y Axis at 0
-                                </MenuLink>
+                                    <input
+                                        type="checkbox"
+                                        className="sr-only"
```

Comment:
@unclejustin I think you misunderstood what I meant – I meant the checkbox input as a _visual_ indicator for _non_-screen-reader-users (therefor the `aria-hidden={true}`), not as _only_ for screen readers as would be with `sr-only`. That's why I removed the SVG checkmark in my code snippet. The screen reader users should already get the `role="menuitemcheckbox"` and its `aria-checked` state announced, ignoring the checkbox input, while sighted users get to see the checkbox input as a purely visual indicator (that of course can be clicked too for toggling thanks to event bubbling, but it's primarily a visual indicator).

Now that I think of it it also should also not be tab-focusable (the menu item would handle changes).
```suggestion
                                        tabIndex={-1}
```

My thought process was that I think it would be good to have a sort of "empty"/"disabled"  state for the checkmark too to indicate that it's a toggle. We could use an empty SVG checkbox, but I was thinking, that's really just trying to _imitate_ a real checkbox, so why not show a real checkbox that any user recognizes 🙂

Reply:
Yeah, I'm trying to stick to the designs as given. My misunderstanding was that the `input` was needed to actually toggle the selected state (spoiler alert: it is not needed). So this was me badly trying to hide it, but keep it.

Reading through your wanting to show an "unchecked" state I totally agree. There isn't very much to visually indicate that this is a toggleable state. I'm going to use the plain checkbox.

cc @AlicjaSuska since we are deviating from your original design. I'd love your input if you disagree with this approach.